src/server/routes/api-routes.js

```diff
@@ -5,4 +5,25 @@ const router = express.Router();
 
 router.use('/items', items);
 
+router.use('/*', (req, res) => {
+  if (res.statusCode === 200 || res.statusCode === 304) {
+    return;
+  }
+  res.status(404).send('API Request > 404 - Page Not Found');
+});
+
+router.use((errHandler, req, res, next) => {
```

Comment:
We should make error handler like that form
4 args are strictly required by express